tests/Module/Fixture/DisplayUserEvents.php

```diff
@@ -85,6 +85,67 @@ public function saveUserEvents(int $owner, array $events)
 
         return ['events' => $events, 'eventsId' => $eventsId];
     }
+
+    public function haveIncrementalEvents(int $owner)
+    {
+        $events = ['incremental_event_first', 'incremental_events_second'];
+        $date = (new \DateTime())->format('Y-m-d');
+
+        foreach(range(0,1) as $i) {
+            $this->laravel->haveRecord(
```

Comment:
I don't understand why you need to add that event two times. Please, add additional test case for creating two events with same event name